usecases/telemetry/payload.go

```diff
@@ -28,11 +28,12 @@ var PayloadType = struct {
 
 // Payload is the object transmitted for telemetry purposes
 type Payload struct {
-	MachineID   strfmt.UUID `json:"machineId"`
-	Type        string      `json:"type"`
-	Version     string      `json:"version"`
-	NumObjects  int64       `json:"objs"`
-	OS          string      `json:"os"`
-	Arch        string      `json:"arch"`
-	UsedModules []string    `json:"usedModules,omitempty"`
+	MachineID      strfmt.UUID `json:"machineId"`
+	Type           string      `json:"type"`
+	Version        string      `json:"version"`
+	NumObjects     int64       `json:"objs"`
+	OS             string      `json:"os"`
+	Arch           string      `json:"arch"`
+	UsedModules    []string    `json:"usedModules,omitempty"`
+	NumCollections int         `json:"numCollections"`
```

Comment:
hmm maybe `collectionsCount` would be a better name?

Reply:
That was the original name I used [here](https://github.com/weaviate/weaviate/pull/7006/commits/ea1a5bceb5755db30ade18f2bb8ca6d3f9e0e2fc#diff-b44781cdcef3725a1644b16a41b82b76cceca662998dfb12d12bc581568b2519R38), but I changed it to match the existing `NumObjects` convention. 

But on second thought, I can just rename both to `*Count`, will still be nonbreaking with the consumer endpoint as long as struct tags remain the same

Reply:
Ok, Done